src/app/modules/time-clock/store/entry.reducer.ts

```diff
@@ -206,6 +206,21 @@ export const entryReducer = (state: EntryState = initialState, action: EntryActi
       };
     }
 
+    case EntryActionTypes.UPDATE_TWO_ENTRIES: {
+      return {
+        ...state,
+        isLoading: true,
+      };
+    }
+
+    case EntryActionTypes.UPDATE_TWO_ENTRIES_FAIL: {
+      return {
```

Comment:
Please see the warnings of Codecov. We are not testing those lines.

Reply:
Thanks, I have tested that line already.